app/controllers/frontend_controller.rb

```diff
@@ -108,14 +109,16 @@ def pick_buyer_or_provider_layout
     end
   end
 
-  def find_service(id = params[:service_id])
-    services = if current_account.try!(:provider?)
+  def all_services
```

Comment:
@hallelujah so with context switcher we need all services in each view, is this acceptable?

Reply:
@hallelujah ping

Reply:
Yes it is though I do not know what happens if you do:

```ruby
all_services.each { |service| service.name }

# Then
find_service
```